documentation/trunking.html

```diff
@@ -0,0 +1,169 @@
+                of this style is that all domains only need to create and maintain a single connection to a single entity: the central hub. This central hub manages and passes through connections between all domains.
+            </p>
+
+            <p>
+                To facilitate a Hub-and-Spoke approach with XMPP domains, the Trunking functionality of Openfire can be used. With this functionality enabled, an Openfire instance can act as the central 'hub' in such a network. In this mode, it can create and accept
+                server-to-server connections with each individual XMPP domain in the network, passing stanzas from one domain to another, and vice versa.
+            </p>
+
+            <h3 id="use-case-data-monitoring-and-filtering">Use Case: Data Monitoring / Filtering</h3>
+            <p>
+                Openfire, in trunking mode, can be deployed as a border filtering device, to check the releasability of data that is exposed from within one network to another.
+            </p>
+            <p>
+                A common scenario is one where a trusted network labels messages that are exchanged with a security label (as defined in <a href="https://xmpp.org/extensions/xep-0258.html">XEP-0258: Security Labels in XMPP</a> for example) that define the confidentiality
+                of the content of the message. An Openfire trunk can be placed between two networks of domains that each have a different security clearance. The trunking Openfire server can filter data (through a <a href="plugin-dev-guide.html">plugin</a>), ensuring that
+                only appropriately labelled messages are exchanged between the XMPP domains, rejecting, for example, messages labelled 'TOP SECRET' sent from an internal XMPP domain to an unrecognized or untrusted XMPP domain.
+            </p>
+
+            <h2 id="setup">Setup</h2>
+            <p>
+                The essence in getting an Openfire instance to act as a Trunking entity in your network is to have all other XMPP domains connect to this server, when they intend to connect to another remote domain. The Trunking entity needs to be configured to accept
+                stanzas for the target domain. This is typically achieved through DNS manipulation. It will then process the data, and pass it on to the target domain.
+            </p>
+
+            <p>
+                The server that is sending the message will attempt to establish a server-to-server connection with the target server. The sending server perform DNS lookups based on the domain name of the target server. The results of this lookup must be manipulated
+                in such a way that the resulting IP address is that of the trunking server. Openfire can facilitate such DNS manipulation, as it allows DNS lookups to be skipped in favor of preconfigured hard-coded values. To configure such values, the
+                property <tt>dnsutil.dnsOverride</tt> can be used.
```

Comment:
Clarify that this applies when Openfire is used as the _sending_ server. 
I know most orgs that might have this use case are likely to have a homogenous estate, but worth calling it out.

Reply:
I've added another sentence to call that out.